tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -74,6 +78,8 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		framework.ExpectNoError(err, "failed to create clientset")
 		asCli, err = asclientset.NewForConfig(config)
 		framework.ExpectNoError(err, "failed to create clientset")
+		genericCli, err = client.New(config, client.Options{Scheme: scheme.Scheme})
```

Comment:
no need to registry k8s scheme?

Reply:
The new `scheme.go` has `k8s.io`, `pingcap.com` and `asts` registered.